### PR TITLE
DOC-61: Render API JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namespaces:
- Render

Other:
- The old Render API is now exposed as the Workload API.

Primary case: https://highfidelity.atlassian.net/browse/DOC-61

Other cases addressed: https://highfidelity.atlassian.net/browse/DOC-52

Note: No executable code changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20658/HighFidelity-Beta-PR16005-e5c58d5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20658/HighFidelity-Quest-Beta-PR16005-e5c58d5.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20658/HighFidelity-Beta-PR16005-e5c58d5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20658/HighFidelity-Beta-Interface-PR16005-e5c58d5.dmg

**Comment by ctrlaltdavid** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20667/HighFidelity-Beta-PR16005-e5c58d5.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20667/HighFidelity-Beta-Interface-PR16005-e5c58d5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20667/HighFidelity-Beta-PR16005-e5c58d5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20667/HighFidelity-Beta-Interface-PR16005-e5c58d5.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20667/HighFidelity-Beta-PR16005-e5c58d5.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20667/HighFidelity-Quest-Beta-PR16005-e5c58d5.apk)

File: interface/src/scripting/RenderScriptingInterface.h [hunk 1]
```diff
@@ -15,13 +15,20 @@
...
  *
  * @namespace Render
  *
  * @hifi-interface
  * @hifi-client-entity
  * @hifi-avatar
+ *
+ * @property {Render.RenderMethod} renderMethod - The render method being used.
+ * @property {boolean} shadowsEnabled - <code>true</code> if shadows are enabled, <code>false</code> if they're disabled.
+ * @property {boolean} ambientOcclusionEnabled - {boolean} <code>true</code> if ambient occlusion is enabled,
```
Comment (jennaingersoll) [inline interface/src/scripting/RenderScriptingInterface.h]:
```suggestion
 * @property {boolean} ambientOcclusionEnabled - <code>true</code> if ambient occlusion is enabled, 
```

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20759/HighFidelity-Beta-PR16005-f974179.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20759/HighFidelity-Beta-Interface-PR16005-f974179.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20759/HighFidelity-Beta-PR16005-f974179.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20759/HighFidelity-Quest-Beta-PR16005-f974179.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20759/HighFidelity-Beta-PR16005-f974179.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20759/HighFidelity-Beta-Interface-PR16005-f974179.exe

File: interface/src/scripting/RenderScriptingInterface.h (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/scripting/RenderScriptingInterface.h]: Done.

**Pull request merged**